src/Image/Image.php

```diff
@@ -39,12 +39,12 @@ public function __construct($data)
     /**
      * @param int $width
      * @param int $height
-     *
+     * @param int $gravity
      * @return Image
      *
      * @throws \Throwable
      */
-    public function crop(int $width, int $height)
+    public function crop(int $width, int $height, int $gravity = Imagick::GRAVITY_CENTER)
```

Comment:
I think it will be a bit more slick to rely on our own constants and not the Imagick ones, just in case we'll want to add support for more adapters or to replace Imagick in the future.

Reply:
Sure